### DISCUSSION TITLE
Unlock hidden console commands

### DISCUSSION THREAD

**Discussion by 64BitDev** [StrataSource/Portal-2-Community-Edition#1199] [Ideas]
Unlock hidden console commands

**Comment by AWildErin** [comment]
I recommend making an issue for that instead of here. This is used for sharing resources with the community (such as instances, models, or whatever.)

**Reply by JJL772** [reply]
There's already an issue

**Reply by AWildErin** [reply]
well okay then, this can be locked since there's no further discussion to be had
